crates/warg/src/lib.rs

```diff
@@ -157,3 +152,49 @@ pub trait Decode: Sized {
 pub trait Encode {
     fn encode(&self) -> Vec<u8>;
 }
+
+/// Helper module for serializing and deserializing timestamps.
+///
+/// This is used over serde's built-in implementation to produce cleaner timestamps
+/// in serialized output.
+mod timestamp {
```

Comment:
Is this actually used somewhere? Afaik, the only timestamps we serialize use the `pbjson_build` implementation of serialize on the pbjson/prost timestamp type(s).

Reply:
It's used in the `Validator` serialization to give a nicer-looking timestamp in the output (`serde_json` serializes `SystemTime` as a struct with verbose names by default).

Reply:
I see, and since `Validator` doesn't have a proto, its allowed to use `SystemTime` directly and not the `google.protobuf.Timestamp` type which has a defined JSON mapping generated by `pbjson_build`.